helm-chart/templates/deployment.yaml

```diff
@@ -89,7 +89,37 @@ spec:
               secretKeyRef:
                 name: {{ include "json-rpc-relay.fullname" . }}
                 key: OPERATOR_KEY_MAIN
-                optional: false                               
+                optional: false
+          - name: DEFAULT_RATE_LIMIT
+            valueFrom:
+              secretKeyRef:
```

Comment:
`secretKeyRef` should be `configMapKeyRef`

Reply:
Fixed